pkg/noderesourcetopology/cache/overreserve.go

```diff
@@ -54,23 +56,24 @@ type OverReserve struct {
 	isPodRelevant          podprovider.PodFilterFunc
 }
 
-func NewOverReserve(cfg *apiconfig.NodeResourceTopologyCache, client ctrlclient.Client, podLister podlisterv1.PodLister, isPodRelevant podprovider.PodFilterFunc) (*OverReserve, error) {
+func NewOverReserve(lh logr.Logger, cfg *apiconfig.NodeResourceTopologyCache, client ctrlclient.Client, podLister podlisterv1.PodLister, isPodRelevant podprovider.PodFilterFunc) (*OverReserve, error) {
 	if client == nil || podLister == nil {
-		return nil, fmt.Errorf("nrtcache: received nil references")
+		return nil, fmt.Errorf("received nil references")
 	}
 
-	resyncMethod := getCacheResyncMethod(cfg)
+	resyncMethod := getCacheResyncMethod(lh, cfg)
 
 	nrtObjs := &topologyv1alpha2.NodeResourceTopologyList{}
 	// TODO: we should pass-in a context in the future
 	if err := client.List(context.Background(), nrtObjs); err != nil {
 		return nil, err
 	}
 
-	klog.V(3).InfoS("nrtcache: initializing", "objects", len(nrtObjs.Items), "method", resyncMethod)
+	lh.V(3).Info("initializing", "objects", len(nrtObjs.Items), "method", resyncMethod)
```

Comment:
maybe change objects to something more descriptive like `nrt count` or just `nrts`

Reply:
using `noderesourcetopologies`